API.md

```diff
@@ -0,0 +1,72 @@
+# SegM8 API
+
+## `class SegM8`
+
+Create an object of type `SegM8` to communicate with a chain of a particular [SegM8 7-segment indicator modules](https://my.amperka.com/modules/SegM8).
+
+### `SegM8(uint8_t pinCS, uint8_t deviceCount = 1)`
+
+Constructs a new SegM8 object that uses the default hardware SPI bus.
+
+- `pinCS`: the *chip select* (also known as *slave select*) pin is used to control the shift-register latch. It is marked **CS** on the SegM8 board. Any GPIO pin of the controller may be used.
+- `deviceCount`: the number of SegM8 modules connected in a daisy-chain. If omitted, defines a single module.
+
+### `SegM8(uint8_t pinCS, uint8_t pinDI, uint8_t pinCLK, uint8_t deviceCount = 1)`
+
+Constructs a new SegM8 object that uses a software SPI bus.
+
+- `pinCS`: the *chip select* (also known as *slave select*) pin is used to control the shift-register latch. It is marked **CS** on the SegM8 module. Any GPIO pin of the controller may be used.
+- `pinDI`: the *data input* pin used to send data to the peripherals. It is marked **DI** on the SegM8 module. In SPI terms known as **MOSI** (Master Out Slave In). Any GPIO pin of the controller may be used.
+- `pinCLK`: the *clock* pin is used to clock pulses that synchronize data transmission generated by the master. It is marked **CLK** on the SegM8 module. Also known as **SCK** in SPI terms. Any GPIO pin of the controller may be used.
+- `deviceCount`: the number of SegM8 modules connected in a daisy-chain. If omitted, defines a single module.
+
+Hardware and software SPI differ in interaction speed. Software SPI runs slower than hardware SPI but hardware SPI works only on specific board pins: **MOSI**, **MISO** (not used in SegM8), and **SCK**.
+
+### `void begin()`
+
+Initializes the given interface, clears the internal buffer, and sets all module segments to the «dark» state.
```

Comment:
In English, typographic quotes are used: “ (open quote, Alt 0147) and ” (close quote, Alt 0148).
```suggestion
Initializes the given interface, clears the internal buffer, and sets all module segments to the “dark” state.
```